bokeh/_json_encoder.py

```diff
@@ -79,4 +79,4 @@ def default(self, obj):
             return self.transform_python_types(obj)
 
 def serialize_json(obj, encoder=BokehJSONEncoder, **kwargs):
-    return json.dumps(obj, cls=encoder, allow_nan=False, **kwargs)
+    return json.dumps(obj, cls=encoder, allow_nan=True, **kwargs)
```

Comment:
Why? Commit 7b741f6 needs explanation. Actually all changes like that need explanation. Did some one made a mistake before and you are fixing this? Did you simply change your mind (then "why?" is even more appropriate)?